executor/union_iter.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package executor
+
+import (
+	"bytes"
+
+	"github.com/pingcap/tidb/kv"
+)
+
+// UnionIter implements kv.Iterator
+type UnionIter struct {
```

Comment:
`UnionIter` is moved to the client-go repo and is now internal
https://github.com/tikv/client-go/blob/master/internal/unionstore/union_iter.go